File: src/main/java/com/company/commitet_jm/entity/Platform.java
package com.company.commitet_jm.entity;

import io.jmix.core.entity.annotation.JmixGeneratedValue;
import io.jmix.core.metamodel.annotation.InstanceName;
import io.jmix.core.metamodel.annotation.JmixEntity;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.util.UUID;

@JmixEntity
@Table(name = "PLATFORM")
@Entity
public class Platform {
    @JmixGeneratedValue
    @Column(name = "ID", nullable = false)
    @Id
    private UUID id;

    @InstanceName
    @Column(name = "NAME")
    private String name;

    @Column(name = "VERSION", length = 20)
    private String version;

    @Column(name = "PATH_INSTALLED")
    private String pathInstalled;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getVersion() {
        return version;
    }

    public void setVersion(String version) {
        this.version = version;
    }

    public String getPathInstalled() {
        return pathInstalled;
    }

    public void setPathInstalled(String pathInstalled) {
        this.pathInstalled = pathInstalled;
    }
}

File: src/main/kotlin/com/company/commitet_jm/entity/Platform.kt
package com.company.commitet_jm.entity

import io.jmix.core.entity.annotation.JmixGeneratedValue
import io.jmix.core.metamodel.annotation.InstanceName
import io.jmix.core.metamodel.annotation.JmixEntity
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import java.util.*

@JmixEntity
@Table(name = "PLATFORM")
@Entity
open class Platform {
    @JmixGeneratedValue
    @Column(name = "ID", nullable = false)
    @Id
    var id: UUID? = null

    @InstanceName
    @Column(name = "NAME")
    var name: String? = null

    @Column(name = "VERSION", length = 20)
    var version: String? = null

    @Column(name = "PATH_INSTALLED")
    var pathInstalled: String? = null
}